src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/LoggerConfiguration.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host.Loggers
+{
+    /// <summary>
+    /// Provides a filter for use with an <see cref="ILogger"/>.
+    /// </summary>
+    [System.CLSCompliant(false)]
+    public class LogCategoryFilter
```

Comment:
Nothing in the .NET Core library does that -- I didn't want to get too off in the weeds with a design that only applies to us. I want to be able to pass this Filter to the `ConsoleLogger` and `DebugLogger` and the other standard loggers, like I do in the sample. There's nothing inherently built-in, but the pattern is to takes a `Func<string, LogLevel, bool>` as a filter, which is what we produce here.

Nothing is stopping us from adding new ways to filter in the future -- this is just matching with the current patterns already in place.